### ISSUE TITLE
Accessibility: Color contrast in section header

### ISSUE THREAD

**Issue by astridu** [LD4P/sinopia_editor#3076]
Make the text in these section headers black.

![image](https://user-images.githubusercontent.com/42216982/135562416-f24bab13-84a1-4d59-a8da-e6500a599f1c.png)

**Comment by astridu** [comment]
@michelleif This whole ticket is irrelevant if dev can implement 3081 and 3083, but if they don't implement those 2, then yes, same applies to the left nav.
